src/plugins/dashboard/public/dashboard_container/component/grid/_dashboard_panel.scss

```diff
@@ -4,19 +4,41 @@
   * .embPanel--editing doesn't get updating without a hard refresh
   */
 
+.dshLayout--editing {
+  .embPanel__hoverActionsAnchor {
+    .embPanel__hoverActionsWrapper {
+      .embPanel__hoverActions {
+        border-color: $euiColorMediumShade;
+        border-style: dashed;
+      }
+    }
+  }
+}
+
 // LAYOUT MODES
 // Adjust borders/etc... for non-spaced out and expanded panels
 .dshLayout-withoutMargins {
-  .embPanel {
+  .embPanel,
+  .embPanel__hoverActionsAnchor {
     box-shadow: none;
+    outline: none;
     border-radius: 0;
   }
 
-  .embPanel__content {
-    border-radius: 0;
+  &.dshLayout--editing {
+    .embPanel__hoverActionsAnchor:hover {
+      outline: 1px dashed $euiColorMediumShade;
+    }
+  }
```

Comment:
<img width="716" alt="Screenshot 2024-09-30 at 9 54 00 PM" src="https://github.com/user-attachments/assets/b82b026f-f7f3-41e2-87c1-b35b360e0db7">
These styles are necessary to add back a border on hover in edit mode when margins are turned off on a dashboard.

<img width="712" alt="Screenshot 2024-09-30 at 9 58 10 PM" src="https://github.com/user-attachments/assets/d425e26f-e6f5-4eed-bc2f-845cbac8177b">
Otherwise it looks like this without a border, and it's hard to distinguish which panel the hover actions are attached to.